.github/workflows/copyright_check.yml

```diff
@@ -0,0 +1,66 @@
+---
+
+name: copyright
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  copyright:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Run check
+        run: |
+          bash - << 'SCRIPT'
```

Comment:
@slry I can't understand, why do you need to run Bash script through this `SCRIPT` file? can't you just directly use the text?

Reply:
@yegor256 this script just do not work in yaml for some reason, so this is a workaround I found

Reply:
@slry it's a weird workaround. I suggest you try to make it work as all other scripts.